docs/releasing.md

```diff
@@ -85,11 +89,12 @@ This triggers two things:
   page, and draft release will be visible on top of the
   [Releases](https://github.com/metal3-io/cluster-api-provider-metal3/releases)
   page.
```

Comment:
Not sure if really necessary, but at least I'd have appreciated if it was highlighted here that one shouldn't click on the generate content in the draft edit view.
I might have been confused because the beta release content was too short. Since that will change this issue might not be relevant anymore.

Reply:
Also, it would be useful to mention how to generate the note using the make command.

Reply:
I can add a note on both. 

Note that with the release note generator improve PR already merged, it will no longer be empty content for beta/RC.